docs/content/docs/help/using-cue.md

```diff
@@ -44,7 +44,7 @@ examples. Of particular note is a comprehension that creates a persistent volume
 claim (PVC) for each pipeline run. A reference to the PVC is added to the
 pipeline run definition by default.
 
-```text
+```txt
```

Comment:
Why not using `cue` here?

Reply:
I'm not seeing it on the supported list:
https://www.getzola.org/documentation/content/syntax-highlighting/

Reply:
Duh... 🤦 I was looking at the GitHub Markdown page, because I used it recently. Sorry about that.